Backend/WebAPI/Controllers/ResetController.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using ServiceLayer.Services;
+using ManagerLayer.UserManagement;
+using DataAccessLayer.Models;
+
+
+namespace WebAPI.Controllers
+{
+    public class ResetController : ApiController
+    {
+
+        //After the user fills in the field with email, this action gets called
+        [HttpPost]
+        [ActionName("SendEmail")]
+        public void SendResetEmail(string email)
+        {
+            ResetService rs = new ResetService();
+            UserManagementManager umm = new UserManagementManager();
+            //Check to see if the email provided is associated with an account
+            if (umm.ExistingUser(email))
+            {
+                //Check to see if the account has a resetID already
+                if (!rs.CheckUserHasResetID(email)) //Don't have a resetID
```

Comment:
Make the logic in this if block the default for every request - no need for the else.